superset/db_engine_specs/databend.py

```diff
@@ -273,7 +276,10 @@ def get_parameters_from_uri(
     ) -> BasicParametersType:
         url = make_url_safe(uri)
         query = url.query
-        if "secure" in query:
```

Comment:
Can we add a migration updating existing URLs to the new format? This way upgrading Superset won't break any existing installations.

Reply:
I was hoping the if...elseif... would cope with URLs of both types, so existing connections using `secure=true/false` would still work anyway